services/hh/service.go

```diff
@@ -170,3 +176,17 @@ func (s *Service) expireWrites() {
 func (s *Service) purgeWrites() {
 	panic("not implemented")
 }
+
+// Nodes returns active nodes for the hinted handoff service.
+func (s *Service) Nodes() []uint64 {
```

Comment:
Why the separate func and not just use the `Node(id)` on meta store directly when processing the queues?